prow/git/v2/client_factory.go

```diff
@@ -169,6 +169,7 @@ func NewClientFactory(opts ...ClientFactoryOpt) (ClientFactory, error) {
 		repoLocks:      map[string]*sync.Mutex{},
 		logger:         logrus.WithField("client", "git"),
 		cookieFilePath: o.CookieFilePath,
+		verifiedRepos:  map[string]bool{},
```

Comment:
nit: we could use `sets.NewString()` (which is internally the same thing), Prow uses that a lot

Reply:
Done